src/components/Governance/index.tsx

```diff
@@ -74,6 +85,10 @@ const GovernancePortal = observer(() => {
       </Grid>
 
       <GovernanceFilterDialog open={showGovernanceFilters} onClose={toggleShowDialog} applyFilter={applyFilter} />
+      <GovernanceRoleList
+        open={showGovernanceRoleList}
```

Comment:
Lets have this look more like a button that leads to a modal rather than something that looks like a link to etherscan